lib/manageiq/api/client/mixins/custom_inspect_mixin.rb

```diff
@@ -0,0 +1,24 @@
+module CustomInspectMixin
+  extend ActiveSupport::Concern
+
+  def inspect
+    res = "#{Kernel.instance_method(:to_s).bind(self).call.chomp!('>')} "
+    attrs = instance_variables - custom_inspect_exclusions
+    res << attrs.map { |attr| "#{attr}=#{custom_inspect_value(attr)}" }.join(", ") << ">"
+  end
+
+  private
+
+  def custom_inspect_exclusions
+    return [] unless self.class.const_defined?(:CUSTOM_INSPECT_EXCLUSIONS)
+    self.class::CUSTOM_INSPECT_EXCLUSIONS.map { |attr| "@#{attr}".to_sym }
```

Comment:
Might be clearer to just have the "@" directly in the CUSTOM_INSPECT_EXCLUSIONS, then no need to manipulate here, and it's clear from the caller perspective that instance variables are the things being excluded.